src/Shift-ClassInstaller-Tests/ShClassInstallerTest.class.st

```diff
@@ -288,6 +287,30 @@ ShClassInstallerTest >> testDuplicatedInstanceVariableInSuperclassMetaclass [
 	self deny: ((superClass class slots collect: [:e | e name]) includes: 'aSlot')
 ]
 
+{ #category : #tests }
+ShClassInstallerTest >> testInstallInSpecificEnvironment [
+
+	| newEnvironment |
+	newEnvironment := SystemDictionary new.
```

Comment:
I would change it into

"newEnvironment := self class environment class new"

to be more generic and not hardcode that we use "SystemDictionary" at the moment